src/routes/safe/components/Apps/components/AppFrame.tsx

```diff
@@ -114,7 +118,17 @@ const AppFrame = ({ appUrl }: Props): ReactElement => {
       clearTimeout(errorTimer.current)
     }
 
-    if (appIsLoading) {
+    if (appIsLoading && currentApp) {
```

Comment:
How is tracking and the app being loaded related?

I would expect a separate hook that depends only on `currentApp` to do the tracking.

Reply:
Does this also ensure that the disclaimer has been agreed to as well?

Reply:
@iamacook Yes. The only way to OPEN THE `<AppFrame />` is if the disclaimer is agreed to.